app/views/pokemons/_form.html.erb

```diff
@@ -27,6 +27,8 @@
     <%= f.input :female_chance %>
   </div>
 
+  <%= f.file_field :image %>
```

Comment:
This is missing some style and all the fields should have labels.

Reply:
Also finish stylizing and give it some label and field.